code/__HELPERS/trait_helpers.dm

```diff
@@ -127,7 +127,7 @@
 		: FALSE)
 #define HAS_TRAIT_NOT_FROM(target, trait, source) (target.status_traits ? (target.status_traits[trait] ? (length(target.status_traits[trait] - source) > 0) : FALSE) : FALSE)
 /// A simple helper for checking traits in a mob's mind
-#define HAS_MIND_TRAIT(target, trait) (HAS_TRAIT(target, trait) || (target.mind ? HAS_TRAIT(target.mind, trait) : FALSE))
+#define HAS_MIND_TRAIT(target, trait) ((target?.mind ? HAS_TRAIT(target.mind, trait) : FALSE))
```

Comment:
I dont see any reason to change the old version?
`(HAS_TRAIT(target, trait)` was needed so it could be called on minds,
`target.mind` has no ? specifically so it runtimes if you pass in no target. IMO it should be runtiming here if you pass in nothing

I think you should change this back